firefox-ios/Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -781,8 +781,13 @@ class BrowserCoordinator: BaseCoordinator,
             navigationController.sheetPresentationController?.detents = [.medium(), .large()]
             navigationController.sheetPresentationController?.prefersGrabberVisible = true
             if isEditing {
-                store.dispatch(ToolbarAction(windowUUID: windowUUID,
-                                             actionType: ToolbarActionType.hideKeyboard))
+                store.dispatchLegacy(
```

Comment:
🚫 `dispatchLegacy` 🚫 I'll remove it in my next PR, but let's not add new ones please! 🙏